src/app/weather-forecast/weather-forecast.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, OnInit} from '@angular/core';
+import { WeatherForecastApiService } from './weatherForecastApiService/weather-forecast-api.service';
+
+@Component({
+    selector: 'app-weather-forecast',
+    templateUrl: './weather-forecast.component.html',
+    styleUrls: ['./weather-forecast.component.scss'],
+})
+export class WeatherForecastComponent implements OnInit {
+    response: object;
+    constructor(private wfas: WeatherForecastApiService) {}
+
+    ngOnInit() {
+        this.retriveWF();
+    }
+    private retriveWF() {
```

Comment:
Bad naming: what is WF? (I know that is is Weather Forecast)
Use a better function name.